backend/src/main/kotlin/org/loculus/backend/utils/FastaReader.kt

```diff
@@ -41,7 +41,7 @@ class FastaReader(inputStream: InputStream) :
                 if (sampleName != null) {
                     break
                 }
-                sampleName = nextLine!!.substring(1)
+                sampleName = nextLine!!.substring(1).substringBefore(" ")
```

Comment:
should this be generalized to all whitespace?

Reply:
done [1935323](https://github.com/loculus-project/loculus/pull/3759/commits/193532345b25ae81c76e4ccb0ddd99bb42070ee8)